src/View/Metadata.php

```diff
@@ -16,6 +17,7 @@ class Metadata
     public function __construct()
     {
         helper('setting');
+        $request = \Config\Services::request();
```

Comment:
Please either move the full namespace class to the top of the file as a `use` statement, or use the `service()` helper method.

Reply:
Actually, it appears this is not being used. Please remove.